components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -112,7 +158,7 @@ export default function EditUserInformationModal({ children }: EditUserInformati
                     <Select
                       placeholder="None Selected"
                       value={gender}
-                      onChange={event => setGender(event.target.value)}
+                      onChange={event => setGender(event.target.value as Gender)}
```

Comment:
Was causing linting error on line 98 so type assertion was used. Type assertion is (safe here)[https://stackoverflow.com/questions/68254890/using-enum-for-event-target-value-in-typescript-and-react] so this should be ok